Header header
float64 Ix
float64 Iy
float64 I